RELEASE-NOTES.txt

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ---
 
+1.70.2
+---
+* [**] Rich Text - Validate link colors [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4542]
```

Comment:
I noticed that the PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/4542 referenced here was closed instead of merged, I understand that the reason for that is that this PR is actually the one that will bring the fix to `develop`, right?

Reply:
It actually auto-closed when I merged the Gutenberg PR 😅 (related to the automation I believe) I'll probably create its own PR to merge it into develop.